[News] 2021-10-04 (#698)
sider/sider-docs: docs/tools/ruby/rails-best-practices.md

---
id: rails-best-practices
title: Rails Best Practices

> sidebar_label: Rails Best Practices (deprecated)

Reviewer: sidebar_label is no longer needed?

Author: No. We will remove it from sidebar and can delete this line.

See also:
https://github.com/sider/sider-docs/pull/631/files#diff-a0383bb15ae59a60a14643df26cd68ac64da9837256a0e73b98cbad6b08397a3

> # Rails Best Practices (unsupported)

Reviewer 2: ```suggestion
# Rails Best Practices
```

How about using the same format?

- https://github.com/sider/sider-docs/blob/master/docs/tools/javascript/tslint.md

Author: Fixed. commit 98311d623ba1035b6ba142819700580672bf7ebf

> :::caution

Reviewer 2: ```suggestion
:::danger
```

How about using the same format?

- https://github.com/sider/sider-docs/blob/master/docs/tools/javascript/tslint.md

Author: Fixed. commit 98311d623ba1035b6ba142819700580672bf7ebf

> We have dropped the support for Rails Best Practices on **September 30, 2021**.

Author: ```suggestion
We have dropped the support for Rails Best Practices on **October 4, 2021**.
```
